schema_based_on_diagram.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE patients (
+	patient_id INTEGER PRIMARY KEY,
+	name VARCHAR,
+	date_of_birth DATE	
+);
+
+CREATE TABLE medical_histories (
+	id INTEGER PRIMARY KEY,
+	admitted_at TIMESTAMP,
+	patient_id INTEGER,
+	status VARCHAR,
+	FOREIGN KEY (patient_id) REFERENCES patients(patient_id)
+);
+
+CREATE TABLE treatments (
+	id INTEGER PRIMARY KEY,
+	type VARCHAR,
+	name VARCHAR
+);
+
+CREATE TABLE invoices (
+	id INTEGER PRIMARY KEY,
+	total_amount DECIMAL,
+	generated_at TIMESTAMP,
+	payed_at TIMESTAMP,
+	medical_history_id INTEGER,
+	FOREIGN KEY (medical_history_id) REFERENCES medical_histories (id)
+);
+
+CREATE TABLE invoice_items (
+	id INTEGER PRIMARY KEY,
+	unit_price DECIMAL,
+	quantity INTEGER,
+	total_price DECIMAL,
+	invoice_id INTEGER,
+	treatment_id INTEGER,
+	FOREIGN KEY (invoice_id) REFERENCES invoices (id),
+	FOREIGN KEY (treatment_id) REFERENCES treatments (id)
+);
```

Comment:
- [ ] Great job creating the tables :clap: . However, there are two missing requirements that you'd need to implement in order to get this milestone complete.
 1. As stated in the requirements, please add indexes for all the foreign keys. :ok_hand: 😊
    #### requirement :point_down: 
![Screenshot from 2023-10-03 14-42-52](https://github.com/iankiema/clinic-database/assets/89228743/8cb2e5a9-2630-4509-bd53-2fc9be4fdede)
2. In the requirements, it is stated that we should create join tables from many-to-many relationships. And we have a `many-to-many` relationship between the `medical_histories` table and the `treatments` table. So I would kindly suggest you create the join table.  :ok_hand: 😊

   #### requirement :point_down: 
![Screenshot from 2023-10-03 14-37-26](https://github.com/iankiema/clinic-database/assets/89228743/9db3ccbb-bed7-426a-8a9d-353e63076952)